test/integration/connect/utils.py

```diff
@@ -0,0 +1,93 @@
+import subprocess
+
+
+class Command:
+    def __init__(self, command, yaml_output=None, lua_output=None):
+        self.command = command
+        if yaml_output is not None:
+            self.exp_output = '---\n- %s\n...\n' % yaml_output
+        elif lua_output is not None:
+            self.exp_output = '%s;' % lua_output
+
+
+commands = [
+        # YAML output
+
+        # valid commands
+        Command('return 666', yaml_output='666'),
+        Command('777', yaml_output='777'),
+        # multiline statement
+        Command('if\ntrue\nthen\nreturn 999\nend', yaml_output='999'),
+        # undefined variable
+        Command('kek', yaml_output='error: \'[string "return kek "]:1: variable \'\'kek\'\' is not declared\''),
+        # syntax error
+        Command('if+1', yaml_output='error: \'[string "if+1 "]:1: unexpected symbol near \'\'+\'\'\''),
+
+        # Lua output
+        Command('\\set output lua', lua_output='true'),
+
+        # valid commands
+        Command('return 666', lua_output='666'),
+        Command('777', lua_output='777'),
+        # multiline statement
+        Command('if\ntrue\nthen\nreturn 999\nend', lua_output='999'),
+
+        # XXX: change it depends on Tarantool version
+        # https://github.com/tarantool/tarantool/commit/ae7e21036b5b3709b5c34d04ddff2b7ac08fabfb
+
+        # # undefined variable
+        # Command('kek', lua_output='"[string \\"return kek \\"]:1: variable \'kek\' is not declared"'),
+        # # syntax error
+        # Command('if+1', lua_output='"[string \\"if+1 \\"]:1: unexpected symbol near \'+\'"'),
+
+        # YAML output again
+        Command('\\set output yaml', yaml_output='true'),
+        Command('return 666', yaml_output='666'),
+    ]
+
+
+def run_commands_in_pipe(project, cmd, commands, exp_connect):
+    process = subprocess.Popen(
+        cmd,
+        cwd=project.path,
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE,
+        stdin=subprocess.PIPE,
+    )
+
+    user_input = '\n'.join([c.command for c in commands]) + '\n'
+
+    out, _ = process.communicate(user_input.encode('utf-8'))
+    out = out.decode('utf-8')
+
+    print(out)
```

Comment:
This print can be removed

Reply:
This print is needed for `pytest -s`, it makes output more verbose. I will add a comment.